exercise-src/react-native/03-creating-a-new-app/03-solution/package.json

```diff
@@ -44,6 +46,6 @@
     "typescript": "5.0.4"
   },
   "engines": {
-    "node": ">=20"
+    "node": ">=18"
```

Comment:
after 3/3, this needs to say `20`

Reply:
Fixed the exercise code and added it to the content, thanks!